pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -141,6 +144,13 @@ def restore_model(self) -> None:
         # restore model state_dict
         self.trainer.training_type_plugin.load_model_state_dict(self._loaded_checkpoint)
 
+        gradients = self._loaded_checkpoint.get("gradients", None)
```

Comment:
what would we call this feature? gradient checkpointing 🤣

Reply:
If the users had gradient accumulation activated.